coffee_core/src/lib.rs

```diff
@@ -22,6 +22,8 @@ pub enum CoffeeOperation {
     /// Search(plugin name)
     Search(String),
     Nurse(bool),
+    /// Tip operation
```

Comment:
```suggestion
    /// Tip operation(plugin, amount)
```
just to keep the convention we have in the file

Reply:
Maybe it is better in a new line, but it is a good suggestion